src/process.cr

```diff
@@ -36,14 +36,20 @@ class Process
   end
 
   # Sends a *signal* to the processes identified by the given *pids*.
+  @[Deprecated("Use #signal instead")]
   def self.kill(signal : Signal, *pids : Int)
     pids.each do |pid|
-      ret = LibC.kill(pid, signal.value)
-      raise RuntimeError.from_errno("kill") if ret < 0
+      signal(signal, pid)
     end
     nil
   end
 
+  # Sends a *signal* to the process identified by the given *pid*.
```

Comment:
```suggestion
  # Sends *signal* to the process identified by *pid*.
```

Reply:
Isn't this just copying the documentation right above it?

Reply:
Yes, but that one's deprecated and will be removed =)
I consider the suggested wording strictly better. The method doesn't send "a" signal but the value of the `signal` argument (which is by convention expressed as italics). That also makes "the given" unnecessary.